CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Contributing to Sparkles App
+
+Thank you for considering contributing to the Sparkles App! We appreciate your help in making this application better. Please follow the guidelines below to get started.
+
+### How to Contribute
+
+#### 1. Fork the Repository
+To contribute to this project, you'll need to create a personal copy of the repository. You can do this by clicking the "Fork" button in the upper right corner of the repository page.
+
+#### 2. Clone Your Fork
+Once you've forked the repository, clone it to your local machine:
+
+```bash
+git clone https://github.com/SparklesIDE/Sparkles-app.git
+cd Sparkles-app
+```
+
+#### 3. Create a New Branch (Optional)
+Create a new branch for your feature or bug fix:
+git checkout -b feature/your-feature-name
+
+#### 4. Make Your Changes
+Implement your changes in your new branch. Be sure to follow the coding style and conventions used in the project.
+
+#### 5. Test Your Changes
+Before you submit your changes, make sure everything works correctly. Run the existing tests and add new tests if applicable.
+
+#### 6. Commit Your Changes
+Once you're satisfied with your changes, commit them with a clear message:
+
+```bash
+git commit -m "Add your detailed description of the change"
+```
+
+#### 7. Push to Your Fork
+Push your changes back to your fork:
+
+1
+```bash
+git push origin feature/your-feature-name
+```
+2 (Default)
+```bash
+git push origin main
+```
+
+> [!Warning]
+> Have you been blocked by Github on your machine? Github's new policy requires an authentication token. Once you get it, you can continue as normal, but with a token. This usually affects when you try to upload to your repository and not to the contributor.
+> https://github.com/settings/tokens/new
+
+#### 8. Open a Pull Request
+Go to the original repository where you want to propose your changes. Click on the "Pull Requests" tab and then click the "New Pull Request" button. Select your branch and submit the pull request.
+
+> [!NOTE]
+>  ### Additional Guidelines
+> - **License**: Please adhere to our [License](license).
```

Comment:
what

Reply:
Just to be aware it is in accordance with the license.